apriltag/src/main/java/edu/wpi/first/apriltag/AprilTag.java

```diff
@@ -54,7 +54,9 @@ public String toString() {
    * @return A RawFrame containing the AprilTag image
    */
   public static RawFrame generate16h5AprilTagImage(int id) {
-    return AprilTagJNI.generate16h5AprilTagImage(id);
+    RawFrame frame = new RawFrame();
+    AprilTagJNI.generate16h5AprilTagImage(frame, frame.getNativeObj(), id);
```

Comment:
Why do we pass both an instance of rawframe and the native object address?

Reply:
We need both on the JNI side, and it’s relatively expensive to get the native address from the JNI side, and cheap to pass both.